extensions/simplelogin/src/api/simplelogin_api.tsx

```diff
@@ -95,23 +95,19 @@ export async function createAlias(values: ParamNewAlias): Promise<AliasResponse
     });
 }
 
-export async function createRandomAlias(note?: string): Promise<AliasResponse> {
+export async function createRandomAlias(note: string): Promise<AliasResponse> {
```

Comment:
This is unrelated to the new feature, I'll explain in the Create Random Alias command.